### PR TITLE
Fix player title overflow

### PR DESCRIPTION
**Pull request by ruisalgado**
There was clearly the intent to "ellipsify" overflowing podcast names in the player title (same as what happens to the show title). However it didn't work in practice due to some css not being applicable to anchors or other inline elements.

Before:
Screenshot 2021-01-01 at 01 53 47
 
After:
Screenshot 2021-01-01 at 01 52 55

Note that there are refactoring opportunities in the player styling - I ignored those as I was initially only trying to get the project up and running when I noticed this issue and wanted to get a quick fix in. I might return to the player in future PRs!

### CODE REVIEW

**Comment by daveajones** [comment]
Thanks for the fix.

@drunkplato do you want to review this?

**Comment by drunkplato** [comment]
Checking into it now

File: ui/src/components/Player/index.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ export default class Player extends React.Component<IProps> {
                             <div className="player-podcast-name">
                                 {episode.feedTitle !== undefined ?
                                     <Link to={`/podcast/${episode.feedId}`} title={episode.feedTitle}>
-                                        from: {episode.feedTitle}
+                                        {`from: ${episode.feedTitle}`}
```
Comment (drunkplato) [inline ui/src/components/Player/index.tsx:145]: It appears that "from:" is now in the line as text. Should only show the subtitle here.

Mainly referring to the images you posted.

Comment (drunkplato) [reply ui/src/components/Player/index.tsx:145]: Okay, sounds good. Thanks for the PR!

**Pull request merged**